### PR TITLE
Fix #55

### LINKED ISSUE #55: Update time is not separated from the Last Report link
## The bug
The last updated time is not separated from the Last Report link.

## Example
Screen Shot 2021-04-08 at 9 20 14 AM
<a href="https://www.reddit.com/r/Mojira/comments/mma9ji/bugtracker_report_21w14a/">Bugtracker Report 21w14a</a>

## Expected behavior
The last updated time is placed on a separate line, or space is included between the Last Report link and the Last updated text.

### PR DESCRIPTION
**Pull request by urielsalis**
#### Checklist
- [ ] Included tests
- [ ] Tested on https://reddit.com/

### CODE REVIEW

File: src/main/kotlin/io/github/mojira/risa/application/ReportGenerator.kt [hunk 1]
```diff
@@ -14,7 +14,7 @@ fun generateReport(
 ): Report = buildString {
     append("[Mojang's Release Post](https://www.minecraft.net/en-us/article/minecraft-snapshot-${currentSnapshot.name})")
     append(" ~ [Last Report](https://www.reddit.com/r/Mojira/comments/${snapshotPosts.getPreviousOf(currentSnapshot)})")
-    append("Last updated: ${Instant.now()}")
+    append(" Last updated: ${Instant.now()}")
```
Comment (violine1101) [inline src/main/kotlin/io/github/mojira/risa/application/ReportGenerator.kt:17]:
```suggestion
    append(" ~ Last updated: ${Instant.now()}")
```
Might make sense to use the same separator as for the previous line.

Comment (violine1101) [reply src/main/kotlin/io/github/mojira/risa/application/ReportGenerator.kt:17]: Also perhaps the date could be formatted a little better? The THH:MM:SS.xxxZ string isn't easy to read.

Comment (urielsalis) [reply src/main/kotlin/io/github/mojira/risa/application/ReportGenerator.kt]: Its the iso format, the one that should cause the less confusion :eyes:

Comment (violine1101) [reply src/main/kotlin/io/github/mojira/risa/application/ReportGenerator.kt]: I'd disagree. If it'd say `2021-04-08 16:00 UTC` or something that would both be easier to read and cause less confusion. ISO is easy for computers to parse, but not for regular users. But perhaps that'd be a change for a different PR.

Comment (urielsalis) [reply src/main/kotlin/io/github/mojira/risa/application/ReportGenerator.kt]: I would put it in a differnt one

**Review by chandler05 — APPROVED** [review]

**Pull request merged**